km_api/know_me/views.py

```diff
@@ -12,6 +12,41 @@
 from know_me import filters, models, serializers
 
 
+class EmergencyContactDetailView(generics.RetrieveUpdateAPIView):
```

Comment:
If we want to be able to delete contacts, this should extend from `generics.RetrieveUpdateDestroyAPIView` and we should add the appropriate test.